pywbem_mock/_wbemconnection_mock.py

```diff
@@ -307,6 +311,19 @@ class must contain variables `provider_type` and
                     self._registry[classname][namespace] = {}
                 self._registry[classname][namespace][provider_type] = provider
 
+            if verbose:
+                _format("Provider {0!A} registered: classes:[{1!A}],  "
+                        "type: {1!A} namespaces:{2!A}",
+                        provider.__class__.__name__,
+                        ", ".join(provider_classnames),
+                        provider_type, ", ".join(namespaces))
+
+            try:
+                provider.post_register_setup(conn)
+            except AttributeError:
+                if verbose:
+                    _format("Register_Provider post_register_setup not found")
```

Comment:
I don't think we should require that a provider has this method. It should be optional: If defined, it is called.

Reply:
Good.  No warning, etc. if missing.